azure-pipelines.yml

```diff
@@ -284,4 +285,70 @@ jobs:
       IntegrationTestsStorageAccountName: $(IntegrationTestsStorageAccountName)
       IntegrationTestsStorageAccountKey: $(IntegrationTestsStorageAccountKey)
     displayName: 'Upload Core Tools build to the storage account for Integration Tests'
-    condition: eq(variables.IsIntegrationBuild, 'true')
\ No newline at end of file
+    condition: eq(variables.IsIntegrationBuild, 'true')
+
+- job: LinuxPackage
+  condition: ne(variables['LinuxPackageBuildTag'], '')
+  timeoutInMinutes: "90"
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+  # Bash v3
+  # Run a Bash script on macOS, Linux, or Windows.
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'  # Specify 'filePath' if you want to use an external script file.
+      script: |
```

Comment:
can we move this to an individual script and run the script as a task instead

Reply:
We will address it as part of other ticket.